### ISSUE TITLE
'DemonAdam' object has no attribute '_set_hyper'

### ISSUE THREAD

**Issue by JOKERKING88** [autasi/demon_sgd#1]
---> 17 model.compile(optimizer=DemonAdam(T= 500 , learning_rate=0.0001), loss='binary_crossentropy', metrics=['accuracy'])
     18 
     19 early = EarlyStopping(monitor="val_acc",mode="max",patience = 50)

<ipython-input-26-cde59c9f1736> in __init__(self, T, learning_rate, beta_1, beta_2, epsilon, amsgrad, name, **kwargs)
     21             **kwargs):
     22         super(DemonAdam, self).__init__(learning_rate, beta_1, beta_2, epsilon, amsgrad, name, **kwargs)
---> 23         self._set_hyper("T", T)
     24 
     25     # see Algorithm 2 in the paper
